src/modules/Dashboard/components/CardImage/CardImage.jsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+
+import { toggleNotesModal, updateModalValue, deleteCurrentImage } from "modules/Dashboard/actions/dashboard";
+import { Button, Card } from "react-bootstrap";
+
+import { faStickyNote, faTrash, faDownload } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { statusIcon } from "./config";
+
+const CardImage = ({ dispatch, imageId, userImage }) => {
+    const handleDeleteImage = () => dispatch(deleteCurrentImage(imageId));
+
+    const handleOpenModal = () => {
+        dispatch(updateModalValue(userImage.notes));
```

Comment:
Use `mapDispatchToProps`, look at [this](https://react-redux.js.org/using-react-redux/connect-mapdispatch).